addon/search/search.js

```diff
@@ -132,7 +132,7 @@
       if (!cursor.find(rev)) return;
     }
     cm.setSelection(cursor.from(), cursor.to());
-    cm.scrollIntoView({from: cursor.from(), to: cursor.to()});
+    cm.scrollIntoView({from: cursor.from(), to: cursor.to()}, 20);
```

Comment:
Without the margin, if a search term is on an early row i.e. Line 1 / 2, after the search cycles back up to the top from the bottom, the highlighted term is off the screen.  Here's a screenshot:

<img width="658" alt="screen shot 2015-07-14 at 10 40 33 am" src="https://cloud.githubusercontent.com/assets/992868/8677816/de44a7a4-2a16-11e5-9a31-cdb1867964b7.png">